CHANGELOG.md

```diff
@@ -1,12 +1,13 @@
 # CHANGELOG
 
-## Next Release
+## v0.5.0 (2022-10-04)
 - New feature: Set expiration time for interactions (how long since it was recorded should an interaction be considered valid)
```

Comment:
Newline between markdown headings and their content please. Markdown doesn't like content right up against a header.